packages/mjml-spacer/src/index.js

```diff
@@ -40,7 +39,7 @@ export default class MjSpacer extends BodyComponent {
         <table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td height="${parseInt(
           height,
           10,
-        )}" style="vertical-align:top;height:${height};">
```

Comment:
I dont'think vertical align is needed here but height is still required

Reply:
Sorry:( Reverted